app/controllers/mixins/ems_common_angular.rb

```diff
@@ -140,6 +140,8 @@ def get_task_args(ems)
         [user, params[:default_password], endpoint_opts]
       when 'ManageIQ::Providers::Lenovo::PhysicalInfraManager'
         [user, password, params[:default_hostname], params[:default_api_port], "token", false]
+      when 'ManageIQ::Providers::Hawkular::MiddlewareManager'
+        [user, password, params[:default_hostname], params[:default_api_port], nil, params[:default_security_protocol]]
```

Comment:
Shouldn't the order be `[..., security_protocol, cert_store]` ?

Reply:
yes ^^ fast fail ^^ But is it tls_ca_certs or cert_store?

Reply:
Rebase with changes

Reply:
I don't know what the key is called but you can set a breakpoint here and try validating creds and puts the params.